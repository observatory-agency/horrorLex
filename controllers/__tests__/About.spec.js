const AboutController = require('../About');

describe('AboutController', () => {
  let about;
  beforeEach(() => {
    about = new AboutController();
  });

  describe('constructor', () => {
    it('should set its templates', () => {
      expect(about.template).toMatchSnapshot();
    });
  });

  describe('get', () => {
    it('should call render with the "get" template', () => {
      const renderMock = jest.fn();
      const resMock = { render: renderMock };
      about.get(null, resMock);
      expect(renderMock).toHaveBeenCalledWith(about.template.get);
    });
  });
});
