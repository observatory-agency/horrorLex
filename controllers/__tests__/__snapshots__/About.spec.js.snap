// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutController constructor should set its templates 1`] = `
Object {
  "get": "about.hbs",
}
`;
